@xen-orchestra/rest-api/src/open-api/oa-examples/network.oa-example.mts

```diff
@@ -0,0 +1,51 @@
+export const networkIds = [
+  '/rest/v0/networks/9fe12ca3-d75d-cfb0-492e-cfd2bc6c568f',
+  '/rest/v0/networks/6b6ca0f5-6611-0636-4b0a-1fb1c8e96414',
+]
+
+export const partialNetworks = [
+  {
+    name_label: 'Host internal management network',
+    id: '9fe12ca3-d75d-cfb0-492e-cfd2bc6c568f',
+    href: '/rest/v0/networks/9fe12ca3-d75d-cfb0-492e-cfd2bc6c568f',
+  },
+  {
+    name_label: 'Lab v2 (VLAN 11)',
+    id: '6b6ca0f5-6611-0636-4b0a-1fb1c8e96414',
+    href: '/rest/v0/networks/6b6ca0f5-6611-0636-4b0a-1fb1c8e96414',
+  },
+]
```

Comment:
In your `fields` field, you document: `nbd, name_label,id`. Ensure you have these properties in your examples